GoRogue.Debugger/Routines/RegionRoutine.cs

```diff
@@ -12,15 +12,24 @@ namespace GoRogue.Debugger.Routines
     [UsedImplicitly]
     internal class RegionRoutine : IRoutine
     {
+        private class CenterOfRotation
+        {
+            internal Point Origin;
+
+            public CenterOfRotation(Point origin)
+            {
+                Origin = origin;
+            }
+        }
```

Comment:
What is the point of this type?  Could it be replaced with a field `Point centerOfRotation` equivalently?  If the type is just to allow it to be added as a component then easily queried via `GetFirst`, perhaps a component tag would be better?

Reply:
The point of this type is so that we can have different, arbitrary points of rotation, so we can have smaller pinwheels in the background going the opposite direction. 

Adding Component Tag

Reply:
Actually, that doesn't work. I'm leaving this as-is